### ISSUE TITLE
Not all files are installed

### ISSUE THREAD

**Issue by aurimasniekis** [symfony/skeleton#8]
Hi, I don't know if I should create this issue in here or on `symfony/flex`. After creating new project using skeleton, not all files and folders are created like in the screencast and etc.

```bash
➜  /tmp composer create-project symfony/skeleton:3.3.x-dev test-flex
Installing symfony/skeleton (3.3.x-dev 19daacfd7ca93335963b7dd0e5ea4e18c228135c)
  - Installing symfony/skeleton (3.3.x-dev 19daacf): Cloning 19daacfd7c from cache
Created project in test-flex
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 22 installs, 0 updates, 0 removals
  - Installing symfony/flex (dev-master 40c8b45): Loading from cache
    Detected auto-configuration settings for "symfony/flex"
    Setting configuration and copying files
  - Installing symfony/stopwatch (dev-master a4cd4b2): Loading from cache
  - Installing symfony/routing (dev-master 2250f38): Loading from cache
  - Installing symfony/polyfill-mbstring (dev-master e79d363): Loading from cache
  - Installing psr/log (dev-master 4ebe3a8): Loading from cache
  - Installing symfony/debug (dev-master 369b129): Loading from cache
  - Installing symfony/http-foundation (dev-master e1831b7): Loading from cache
  - Installing symfony/event-dispatcher (dev-master 82ed8cb): Loading from cache
  - Installing symfony/http-kernel (dev-master 31ff495): Loading from cache
  - Installing symfony/finder (dev-master f56117f): Loading from cache
  - Installing symfony/filesystem (dev-master 28dcd11): Loading from cache
  - Installing psr/container (dev-master b7ce3b1): Loading from cache
  - Installing symfony/dependency-injection (dev-master a8e4dea): Loading from cache
  - Installing symfony/config (dev-master 3e08f33): Loading from cache
  - Installing symfony/class-loader (3.4.x-dev cf3e7ac): Loading from cache
  - Installing psr/simple-cache (dev-master 753fa59): Loading from cache
  - Installing psr/cache (dev-master 78c5a01): Loading from cache
  - Installing symfony/cache (dev-master b6c47b0): Loading from cache
  - Installing doctrine/cache (dev-master 0da649f): Loading from cache
  - Installing symfony/framework-bundle (3.4.x-dev bb390c7): Loading from cache
    Detected auto-configuration settings for "symfony/framework-bundle"
    Enabling the package as a Symfony bundle
  - Installing symfony/yaml (3.4.x-dev d7906a0): Loading from cache
  - Installing symfony/dotenv (3.4.x-dev 80f0687): Loading from cache
Writing lock file
Generating autoload files

➜  /tmp
```

```bash
➜  test-flex ls -la
total 120
drwxr-xr-x   8 aniekis  wheel    272 May 18 15:15 .
drwxrwxrwt  86 root     wheel   2924 May 18 15:15 ..
-rw-r--r--   1 aniekis  wheel    332 May 18 15:15 .env
-rw-r--r--   1 aniekis  wheel    332 May 18 15:15 .env.dist
-rw-r--r--   1 aniekis  wheel   1121 May 18 15:15 composer.json
-rw-r--r--   1 aniekis  wheel  48302 May 18 15:15 composer.lock
drwxr-xr-x   3 aniekis  wheel    102 May 18 15:15 etc
drwxr-xr-x   7 aniekis  wheel    238 May 18 15:15 vendor
➜  test-flex```

**Comment by chalasr** [comment]
See https://github.com/symfony/flex/issues/88

**Comment by stof** [comment]
that's because FrameworkBundle 3.4.x-dev has been created now, but the recipes have not yet been updated to tell the flex server that they support 3.4

**Comment by vladimmi** [comment]
> They must follow the vendor/package/version/ directory structure, where version is the minimum version supported by the recipe.
Am I missing something or recipes for 3.3 should support 3.4 as well, because 3.3 in recipe is **minimum**, not a strict version?

**Comment by stof** [comment]
@vladimmi this is something we discussed with @fabpot yes. We all think it should be the expected behavior of the Flex server. But as said in all communications about it, Flex is still experimental. So this is not yet how thinks work

**Comment by fabpot** [comment]
Will be fixed soon, see symfony/recipes#70
